src/ipelib/IPE_Electrodynamics_Class.F90

```diff
@@ -205,7 +206,14 @@ SUBROUTINE Update_IPE_Electrodynamics( eldyn, grid, forcing, time_tracker, plasm
 
       CALL eldyn % Dynamo_Wrapper(grid, forcing, time_tracker, plasma, mpi_layer )
       IF( mpi_layer % rank_id == 0 )THEN
-        print *,'Dynamo E field ', int(time_tracker % elapsed_sec / 60), ' mins UT'
+        elapsed_hours = int(time_tracker % elapsed_sec / 3660)
+        elapsed_minutes = (int(time_tracker % elapsed_sec / 60)) - (elapsed_hours * 60)                                                              
+       write(6,*) '*********************************'
+        print *,'peee Calling Dynamo E field, elapsed time = ', elapsed_hours, elapsed_minutes, &
```

Comment:
"peee " can be removed -- could also calculated elapsed_hours and elapsed_minutes directly in the write rather than allocating space for those in memory, but this isn't a big deal.